src/Microsoft.AspNetCore.Authentication.OpenIdConnect/OpenIdConnectHandler.cs

```diff
@@ -320,6 +320,14 @@ protected virtual Task<bool> HandleSignOutCallbackAsync()
                 Scope = string.Join(" ", Options.Scope)
             };
 
+            string maxAge = OpenIdConnectHelper.GetMaxAgeParameterFromTimeSpan(Options.MaxAge);
+            // Add the 'max_age' parameter to the authentication request if MaxAge is at least one second.
+            // See http://openid.net/specs/openid-connect-core-1_0.html#AuthRequest
+            if (maxAge != null)
+            {
+                message.MaxAge = maxAge;
```

Comment:
OpenIdConnectMessage allows assigning null, you don't need to null check here. You can inline all of this right after Scope.